website/js/index.js

```diff
@@ -0,0 +1,20 @@
+const fileInput = document.getElementById('file-input');
+document.getElementsByTagName("h1")[0].innerHTML = "main";
+fileInput.addEventListener('change', (e) => displayImage(e.target.files));
+
+function displayImage(fileList) {
+  document.getElementsByTagName("h1")[0].innerHTML = "attempting to display";
+  let file = null;
+
+  for(let i = 0; i < fileList.length; i++) {
```

Comment:
You should be able to remove this check as the HTML only accepts images - e.g. `accept="image/*"`

Reply:
Hi Matt,

I agree with not having to explicitly check the image but wouldn't the check ensure the future usability and readability of the code? (The reader can see that only images can be displayed). I also feel that having the
`accept="image/*"` 
is more for the UX feature to help guide the user what file to choose. We can do either way but I just want to know the benefits of not checking? It could be that it just results in less code?

Reply:
Good question. My rationale here: if you enforce the MIME type as `image/*` in the HTML, then you don't need to validate the MIME type in JavaScript, as the user can't have (without circumventing the limitation) uploaded a file that isn't an image.

Thus: simpler code (easier to reason; less likely to fail) and better UX. Right now, your code would just break out of the loop and do nothing - no error message, etc.